### PR TITLE
Add EQL track

### PR DESCRIPTION
**Pull request by matriv**
Add a new track to benchmark EQL performance.

### CODE REVIEW

File: eql/track.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "Tracker-generated track for eql",
```
Comment (dliappis) [inline eql/track.json:4]: Let's change this description to something useful.

**Review by dliappis — APPROVED** [review]
I left a comment about changing the (auto generated) description.

~Also let's mention in the README that this track is designed to be executed against Elasticsearch master only~
Offline I understood this will work also with 7.10+ so after merging we should backport to the `7` branch.

Otherwise LGTM, no need for another review cycle.

**Comment by matriv** [comment]
@dliappis That's correct we want it in >= 7.10

File: eql/track.json (hunk 1 above)
Comment (matriv) [reply eql/track.json]: Sure, done!

**Pull request merged**